bin/openwhisk-standalone-config/runtimes.json

```diff
@@ -1,6 +1,12 @@
 {
   "runtimes": {
     "nodejs": [
+      {
+        "kind": "blackbox",
+        "image": {
+          "name": "blackbox"
+        }
+      },
```

Comment:
Thanks! I think it's best to update this script to add this entry statically, since that's how we generate this json: https://github.com/adobe/aio-cli-plugin-app/blob/master/bin/openwhisk-standalone-config/get-runtimes.js

related: https://github.com/adobe/aio-cli-plugin-app/pull/741